src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleared = sourceString.split(';');
```

Comment:
[CODE STYLE] - The variable name 'cleared' is not descriptive of its contents or purpose. Consider using a more descriptive name, such as 'styleDeclarations', to improve code readability.